### PR TITLE
PS Video Library

### PR DESCRIPTION
**Pull request by nancymic2**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

- Create video page to consolidate resources
- https://pr-6370-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/videos/
-

### CODE REVIEW

**Comment by nancymic2** [comment]
FYI Alexandra said to use my name in the byline but is that ok?

**Review by anusmitaray — APPROVED** [review]

File: site/en/docs/privacy-sandbox/videos/index.md [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+Keep in mind that while the concepts discussed in these videos are still relevant, some technical details and code will have changed.
+
+Refer to the linked documentation for the latest details.
+{% endAside %}
+
+## What is the Privacy Sandbox?
+
+{% Columns %}
+{% Column %}
+<iframe width="280" src="https://www.youtube.com/embed/NKz5oT6kXI4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Comment (rowan-m) [inline site/en/docs/privacy-sandbox/videos/index.md:25]: Preferable to use the specific YouTube embed, I think.
https://developer.chrome.com/docs/handbook/how-to/add-a-youtube-embed/

File: site/en/docs/privacy-sandbox/videos/index.md [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+subhead: >
+  Find overviews, walkthroughs, and conceptual discussions in these Privacy Sandbox videos.
+description: >
+  Privacy Sandbox video library
+date: 2023-06-26
+authors:
+  - nmichell
+---
+
+If you're looking for Privacy Sandbox resources, check out the videos below. Some are recorded Privacy Sandbox office hours. You can find upcoming office hours on the [events page](/docs/privacy-sandbox/events/).
```
Comment (rowan-m) [inline site/en/docs/privacy-sandbox/videos/index.md:13]: I think the opener here should be more along the lines of describing the selection of videos, as in from past events, series, and office hours. We can probably assume that if the reader is on this page, then they are definitely looking for Privacy Sandbox resources. 😁

File: site/en/docs/privacy-sandbox/videos/index.md [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+<iframe width="280" height="158" src="https://www.youtube.com/embed/VVgtGkRtVPU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+<br>
+Refer to the documentation for the latest details: <a href="/docs/privacy-sandbox/overview/">What is the Privacy Sandbox?</a>
+{% endColumn %}
+{% Column %}
+<strong>May 12, 2022. Run time: 16:16</strong>
+
+<strong>Latest docs:</strong> [What is the Privacy sandbox?](/docs/privacy-sandbox/overview/)
+
+At Google I/O 2022, Rowan Merewood from the Chrome team spoke about the Privacy Sandbox, the pros and cons of third-party content, and how the Privacy Sandbox addresses these issues.
```
Comment (rowan-m) [inline site/en/docs/privacy-sandbox/videos/index.md:62]: Where names are mentioned and there's an author profile, that would be good to link.

**Review by rowan-m — CHANGES_REQUESTED** [review]

File: site/en/docs/privacy-sandbox/videos/index.md (hunk 2 above)
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/videos/index.md]: PTAL @rowan-m

**Review by rowan-m — APPROVED** [review]
LGTM, thanks!

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**